monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/JobsFolderController.java

```diff
@@ -86,22 +93,88 @@
+                        jobDetails.put("name", job.getName());
+                        jobDetails.put("filename", file.getName());
+                        jobDetails.put("repository_path", file.getQualifiedPath());
+                        jobDetails.put("metadataProperties", jobMetadataProperties);
+                        final List<Map<String, Object>> descriptors = new ArrayList<Map<String, Object>>();
+
+                        if (jobContext instanceof DataCleanerJobContext) {
+                            final DataCleanerJobContext dcContext = (DataCleanerJobContext) jobContext;
+
+                            final AnalysisJob analysisJob = dcContext.getAnalysisJob();
+                            final List<AnalyzerJob> analyzerJobs = analysisJob.getAnalyzerJobs();
+                            for (AnalyzerJob analyzerJob : analyzerJobs) {
+                                final Map<String, Object> jobComponent = new HashMap<String, Object>();
+                                jobComponent.put("type", "analyzer");
+                                jobComponent.put("descriptor", analyzerJob.getDescriptor().getDisplayName());
+                                jobComponent.put("metadataProperties", analyzerJob.getMetadataProperties());
+                                descriptors.add(jobComponent);
+                            }
+
+                            List<TransformerJob> transformerJobs = analysisJob.getTransformerJobs();
+                            for (TransformerJob transformerJob : transformerJobs) {
+                                final Map<String, Object> jobComponent = new HashMap<String, Object>();
+                                jobComponent.put("type", "transformer");
+                                jobComponent.put("descriptor", transformerJob.getDescriptor().getDisplayName());
+                                jobComponent.put("metadataProperties", transformerJob.getMetadataProperties());
+                                descriptors.add(jobComponent);
+                            }
+
+                            List<FilterJob> filterJobs = analysisJob.getFilterJobs();
+                            for (FilterJob filterJob : filterJobs) {
+                                final Map<String, Object> jobComponent = new HashMap<String, Object>();
+                                jobComponent.put("type", "filter");
+                                jobComponent.put("descriptor", filterJob.getDescriptor().getDisplayName());
+                                jobComponent.put("metadataProperties", filterJob.getMetadataProperties());
+                                descriptors.add(jobComponent);
+                            }
+                        }
+                        jobDetails.put("descriptors", descriptors);
+                        result.add(jobDetails);
+                    }
+                }
+            }
+        }
+        return objectMapper.writeValueAsString(result);
```

Comment:
I had an error regarding the mapping of the methods that both return JSON. I was reluctant to add a new mapping to the new method. Nevertheless I have added a path to the mapping and everything works fine now. 

´´´
http://localhost:8080/DataCleaner-monitor/repository/demo/jobs/filter?property=Category&value=Enrichment```